### PR TITLE
Module1:Mobile menu

### PR DESCRIPTION
**Pull request by hammaazarok**
* add linter for HTML,CSS, Javascript
* add main.js to manipulate DOM
* restyle && rearrange document structure to implement mobile menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,21 +11,28 @@
...
-            <a href="#">
-                <p>Hamma Azarok</p>
+            <a href="#" class="hamburger">
+                <span class="src-only" hidden="true">Open  main menu</span>
+                <i class="fas fa-bars"></i>
             </a>
-            <nav> <img src="img/Enabled.png" alt="Logo"></nav>
-        </header>
+        </nav>
+    </header>
```
Comment (Mre55) [inline index.html:34]: Kindly, stick to the design provided for the padding of the `nav` bar it doesn't have any padding around it please edit that.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

File: index.html [hunk 2]
```diff
@@ -11,21 +11,28 @@
...
+    
         <section class="header">
             <div class="header-content">
                 <h1 class="header-text">I’m Hamma
```
Comment (Mre55) [inline index.html:38]: Please stick to the design and make your background blur as of the design when the mobile menu appears.

| Your Design | Figma Design |
|-------------|---------------|
|image | image |

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @hammaazarok,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 

_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -11,22 +11,28 @@
...
+                <li class="nav-item"><a href="#about" class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#contact" class="nav-link">Contact</a></li>
             </ul>
-        </nav>
-        <header class="nav-bar">
-            <a href="#">
-                <p>Hamma Azarok</p>
+            <a href="#" class="hamburger">
+                <span class="src-only" hidden="true">Open  main menu</span>
+                <i class="fas fa-bars"></i>
```
Comment (tresorsawasawa) [inline index.html:31]: - [ ] Good job by adding the blur in the background of the mobile menu. But as the previous reviewer mentioned in the previous comment to stick the background of the mobile menu to the one from Fimga, I think you missed one detail on the height. I just noticed that from the Figma template the background of the mobile menu covers the entire page i.e. from header to footer. I think once you fixed this issue your milestone will be completed. I know you can do it.

**Pull request merged**